### DISCUSSION TITLE
Feature bounty

### DISCUSSION THREAD

**Discussion by gam-ragnar** [mediacms-io/mediacms#616] [Feature Requests]
Feature bounty anyone? Lets make a list of critical features and try collective funding.

**Comment by Dan1ell** [comment]
How would that work exactly?

**Comment by gam-ragnar** [comment]
Hi @Dan1ell 
State how much you willing to donate to have a feature implemented, If you need others to chip in ask. And dev guys can either take up the offer or we ask at bountysource for a freelancer to do it.

**Comment by NoahGWood** [comment]
Might want to try something like
https://issuehunt.io/

I'm bored and looking for stuff I can make for a few bucks

**Comment by mgogoulos** [comment]
> Might want to try something like
> https://issuehunt.io/
> 
> I'm bored and looking for stuff I can make for a few bucks

Hi @NoahGWood , how popular is this service? can you share any examples on projects using it so we get a better idea? Thanks
